spanner/google/cloud/spanner/__init__.py

```diff
@@ -18,6 +18,13 @@
 import pkg_resources
 __version__ = pkg_resources.get_distribution('google-cloud-spanner').version
 
+from google.cloud.spanner._helpers import BOOL_PARAM_TYPE
```

Comment:
I would like to move in the direction of adding a `types` module for everything; I had not done it for Spanner yet.

@tseaver Any objections to my doing that if I can do it today or tomorrow?